### ISSUE TITLE
BUG: Batch BCF Conversion Assumes XRF

### ISSUE THREAD

**Issue by jkabel** [Bruker-Amics/bugs_features#506]
The batch BCF conversion feature seems to assume BCFs are XRF data without checking, and Process 3.3 won't match XRF datasets against SEM libraries.

Individual BCF opening/conversion seems to work just fine.

**Comment by AlexeiDolgolyov** [comment]
@ChiLyMan, can you, please, provide more details about it for development team?

**Comment by ChiLyMan** [comment]
@AlexeiDolgolyov I'm going to close this because user no longer works there and we will not get anymore details on this bug. Will need to wait till someone else encounters this issue.

**Comment by jkabel** [comment]
I didn't die or anything though.

I collected a bunch of maps using Esprit Jobs on an SEM. As an experiment, I decided to feed the resulting BCFs into AMICS to look for mineral identifications.

When individually importing a BCF, AMICS would recognize that it was an SEM dataset and tag it as such for AMICS to apply SEM-EDS libraries.

When the maps were batch imported, AMICS presumably didn't check and would default to tagging them as XRF datasets. Process would then only allow matching the resulting datasets against XRF libraries.

**Comment by ChiLyMan** [comment]
Hey Jacob, thanks for letting us know you are alive! Don't forget to let us know where you are to stay in contact.
OK now we can understand this better, we can move this into for later version which will have a new material management interface.
    On Thursday 12 September 2024 at 12:01:59 pm GMT-5, jkabel ***@***.***> wrote:  
 
 


I didn't die or anything though.

I collected a bunch of maps using Esprit Jobs on an SEM. As an experiment, I decided to feed the resulting BCFs into AMICS to look for mineral identifications.

When individually importing a BCF, AMICS would recognize that it was an SEM dataset and tag it as such for AMICS to apply SEM-EDS libraries.

When the maps were batch imported, AMICS presumably didn't check and would default to tagging them as XRF datasets. Process would then only allow matching the resulting datasets against XRF libraries.
